platform/src/common/components/Map/components/Sidebar.js

```diff
@@ -229,7 +232,7 @@ const WeekPrediction = ({ siteDetails, currentDay, airQualityReadings, weekDays
 
   return (
     <div className='relative' ref={dropdownRef}>
-      <div className='mb-5 relative'>
+      {/* <div className='mb-5 relative'>
```

Comment:
Do we plan to use this commented code in the near future?

Reply:
Yes it's going to be used later